cibuildwheel/docker_container.py

```diff
@@ -30,6 +30,9 @@ class DockerContainer:
     bash_stdout: IO[bytes]
 
     def __init__(self, docker_image: str, simulate_32_bit: bool = False, cwd: Optional[PathOrStr] = None):
+        if not docker_image:
+            raise RuntimeError("Must have a non-empty docker image to run.")
```

Comment:
To nit-pick the most minor of minor points... 
```suggestion
            raise ValueError("Must have a non-empty docker image to run.")
```